test/testdata/lsp/formatting.rb.document-formatting-rubyfmt.exp

```diff
@@ -1,5 +0,0 @@
-# typed: false
-class MyMalformedClass
-  def bad_whitespace
-  end
-end
```

Comment:
Taking the existing test files out for now -- in a later PR, I'll be adding proper tests, but we'll have to stub out the `rubyfmt` executable with something else, so these tests won't be applicable for that (since they won't actually do any formatting).